disco_aws_automation/disco_vpc.py

```diff
@@ -224,18 +231,20 @@ def _create_new_meta_networks(self):
 
     def find_instance_route_table(self, instance):
         """ Return route tables corresponding to instance """
-        rt_filter = self.vpc_filter()
-        rt_filter["route.instance-id"] = instance.id
-        return self.vpc.connection.get_all_route_tables(filters=rt_filter)
+        rt_filter = []
+        rt_filter.append(self.vpc_filter())
```

Comment:
Sure, I'll create a helper method like the following:
`
def create_filters(filter_dict):
    filters = []
    for key in filter_dict.keys():
        filters.append({'Name': key, 'Values': filter_dict[key]})
    return filters
`
`create_filters({:route.instance-id": [instance.id]})`